dist/tools/flake8/check.sh

```diff
@@ -45,6 +47,18 @@ ${FLAKE8_CMD} --version &> /dev/null || {
 
 ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES})
 
+if github_annotate_is_on; then
+    grep "^.\+:[0-9]\+:" "${ERRORS}" | while read line; do
```

Comment:
Ah! `${ERRORS}` of course is a string, not a filename

```suggestion
    echo "${ERRORS}" | grep "^.\+:[0-9]\+:" | while read line; do
```